library/core/src/str/mod.rs

```diff
@@ -84,8 +84,8 @@ pub trait FromStr: Sized {
     /// when the string is ill-formatted return an error specific to the
     /// inside [`Err`]. The error type is specific to implementation of the trait.
     ///
-    /// [`Ok`]: ../../std/result/enum.Result.html#variant.Ok
-    /// [`Err`]: ../../std/result/enum.Result.html#variant.Err
+    /// [`Ok`]: Result::Ok
+    /// [`Err`]: Result::Err
     ///
```

Comment:
These are in scope from the prelude, so they don't need reference links at all :)
```suggestion
```